airflow/providers/google/cloud/example_dags/example_bigquery_queries.py

```diff
@@ -125,6 +126,39 @@
         )
         # [END howto_operator_bigquery_select_job]
 
+        extract_job = BigQueryInsertJobOperator(
+            task_id="extract_to_gcs_job",
+            configuration={
+                "extract": {
+                    "sourceTable": {
+                        "projectId": PROJECT_ID,
+                        "datasetId": DATASET_NAME,
+                        "tableId": TABLE_1,
+                    },
+                    "destinationUris": ["gs://example_bucket/dump/table.*.csv.gz"],
```

Comment:
Could you put bucket it in the environment variable? Avoiding hard coding will allow to run system tests

Reply:
This bucket should also be created in system test

Reply:
@turbaszek where can I find the code for that? is it outside of this repo?

Reply:
System tests for BigQuery: https://github.com/apache/airflow/blob/master/tests/providers/google/cloud/operators/test_bigquery_system.py
Docs for system tests:
https://github.com/apache/airflow/blob/master/TESTING.rst#airflow-system-tests

Reply:
i found one for the gcs example dag, can I just use that one?

Reply:
No because this will require gcs test to be run before running bq test. Creating a bucket is simple like that:
https://github.com/apache/airflow/blob/7c5cdcf30b2967c3e9f3652e58021a8553be1029/tests/providers/google/cloud/operators/test_bigquery_system.py#L38

also, if you need to a file to be uploaded to this bucket you can do:
https://github.com/apache/airflow/blob/7c5cdcf30b2967c3e9f3652e58021a8553be1029/tests/providers/google/cloud/operators/test_dataproc_system.py#L56